tests/schema/tests.py

```diff
@@ -1545,6 +1547,63 @@ def test_check_constraints(self):
         if not any(details['columns'] == ['height'] and details['check'] for details in constraints.values()):
             self.fail("No check constraint for height found")
 
+    @skipUnlessDBFeature('supports_column_check_constraints')
+    def test_remove_field_check_does_not_remove_custom_checks(self):
+        # Create table
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+
+        # Add the custom check constraint
+        constraint = CheckConstraint(check=Q(height__gte=0), name='author_height_gte_0_check')
+        custom_constraint_name = constraint.name
+
+        try:
+            Author._meta.constraints = [constraint]
+            with connection.schema_editor() as editor:
+                editor.add_constraint(Author, constraint)
+
+            # Ensure the constraints exist
+            constraints = self.get_constraints(Author._meta.db_table)
+            self.assertIn(custom_constraint_name, constraints)
+            self.assertEqual(
```

Comment:
This would be more readable and consistent with our indentation style with something like:
```
foo_constraints = [
    name  for name, details in constraints.items()
    if details['columns'] == ['name'] and details['unique'] and name != custom_constraint_name]
]
self.assertEqual(len(foo_constraints), 1)
```
(choosing a different name than "foo"

Reply:
changed